mass_mailing_event_registration_exclude/models/mailing_search_exclude_mixin.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Tenativa - Eduardo Ezerouali
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailingSearchExcludeMixin(models.AbstractModel):
+    _name = "mailing.search.exclude.mixin"
+    _description = ""
```

Comment:
Can you define a description there to avoid the error “https://github.com/OCA/mass-mailing/actions/runs/18005389174/job/51224321824?pr=6#step:8:324” in the CI?